.circleci/config.yml

```diff
@@ -34,6 +34,9 @@ version: 2
 
           echo "Using the $branch branch"
 
+          # If web directory already exists, empty it out
+          mkdir -p web
+          rm web/*
           # Uncompress the files to the "web" folder
           curl "$root/$branch/$file" -o web.tar.gz
           mkdir web
```

Comment:
This line should be removed, since the directory is already created.

Reply:
Ah, yeah.